NorthstarDLL/logging.cpp

```diff
@@ -118,16 +120,31 @@ void CustomSink::custom_log(const custom_log_msg& msg)
 	custom_sink_it_(msg);
 }
 
+void CloseConsole()
+{
+	spdlog::info("Closing console now.");
+	TerminateProcess(processInfo.hProcess, 0);
+}
+
 void InitialiseConsole()
 {
-	if (AllocConsole() == FALSE)
-	{
-		std::cout << "[*] Failed to create a console window, maybe a console already exists?" << std::endl;
-	}
-	else
+	bool hasAttached = AttachConsole(-1);
```

Comment:
I need to check how this interacts with the docker image before this is merged.

Reply:
Report back with your findings. More than happy to help if this creates issues.

Reply:
Confirmed working under docker

Reply:
Can I see the first 50 or so lines of the log to check a few things?